tools/mo/openvino/tools/mo/convert_impl.py

```diff
@@ -413,14 +413,16 @@ def prepare_ir(argv: argparse.Namespace):
     return graph, ngraph_function
 
 
+def read_model(fem: FrontEndManager, path_to_xml: str):
+    fe = fem.load_by_framework(framework="ir")
+    function = fe.convert(fe.load(path_to_xml))
+    return function
+
+
 def emit_ir(graph: Graph, argv: argparse.Namespace, non_default_params: dict):
     # We have to separate fe object lifetime from fem to
     # avoid segfault during object destruction. So fe must
     # be destructed before fem object explicitly.
```

Comment:
this comment is left and relates to the function which you moved.

Reply:
moved this comment to the function `read_model` where is was meant to be